### ISSUE TITLE
Music queue/playback issue

### ISSUE THREAD

**Issue by marie-haskova** [esmBot/esmBot#309]
### Command that caused the error

/music play <query>, then /play skip

### Self-hosted instance?

No

### Error file

```
Message: TypeError: Cannot read properties of undefined (reading 'host')
    at SkipCommand.run (file:///home/esmBot/esmBot/commands/music/skip.js?v=107:11:16)
    at MusicAIOCommand.run (file:///home/esmBot/esmBot/commands/music/music.js?v=100:18:34)
    at default (file:///home/esmBot/esmBot/events/interactionCreate.js:27:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Stack Trace: TypeError: Cannot read properties of undefined (reading 'host')
    at SkipCommand.run (file:///home/esmBot/esmBot/commands/music/skip.js?v=107:11:16)
    at MusicAIOCommand.run (file:///home/esmBot/esmBot/commands/music/music.js?v=100:18:34)
    at default (file:///home/esmBot/esmBot/events/interactionCreate.js:27:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

### Additional context

bot joins the vc, doesnt play music, and when `/music skip` is entered it throws this error

**Comment by TheEssem** [comment]
Did the bot ever send a message back with the song info?

**Comment by Ikilledmywife** [comment]
it did, I submitted a search query, it returned the name of the youtube video just fine.

*Your tune* `My Chemical Romance - Make Room!!!! [Official Audio]` *has been added to the queue!*

**Comment by TheEssem** [comment]
Did it ever post an embed that looked anything like this? If not, was this the first song queued or did anyone else play any other songs beforehand at some point?

![image](https://user-images.githubusercontent.com/21112968/188284786-f51f017a-706c-4572-bf26-3b1e01cea635.png)

**Comment by Ikilledmywife** [comment]
it just stopped doing anything when it got to that specific song in the queue

**Comment by TheEssem** [comment]
Should be fixed by 0c08422
